androidApp/src/main/java/com/mbta/tid/mbta_app/android/stopDetails/TripStopRow.kt

```diff
@@ -78,32 +80,28 @@ fun TripStopRow(
                 horizontalArrangement = Arrangement.spacedBy(0.dp),
                 verticalAlignment = Alignment.CenterVertically,
             ) {
-                if (showElevatorAccessibility) {
-                    Icon(
-                        modifier =
-                            Modifier.height(24.dp)
-                                .padding(
-                                    start = if (stop.stop.isWheelchairAccessible) 6.dp else 3.dp
-                                ),
-                        painter =
-                            if (stop.stop.isWheelchairAccessible) {
-                                painterResource(R.drawable.wheelchair_accessible)
-                            } else {
-                                painterResource(R.drawable.elevator_alert)
-                            },
-                        contentDescription = null,
-                        tint = Color.Unspecified
-                    )
+                Row(
+                    Modifier.padding(start = 6.dp).width(28.dp),
+                    verticalAlignment = Alignment.CenterVertically,
+                    horizontalArrangement = Arrangement.Center
+                ) {
+                    if (showElevatorAccessibility && stop.activeElevatorAlerts(now).isNotEmpty()) {
+                        Icon(
+                            modifier = Modifier.height(24.dp).testTag("elevator_alert"),
+                            painter = painterResource(R.drawable.elevator_alert),
+                            contentDescription = null,
+                            tint = Color.Unspecified
+                        )
+                    } else if (showElevatorAccessibility && stop.stop.isWheelchairAccessible) {
+                        Icon(
```

Comment:
If you're overriding the size and removing the tint, you may as well use `Image` instead of `Icon` - `Icon` is mostly useful if you want to render an icon at the default Android icon size and in the current content color.